### DISCUSSION TITLE
Is it possible to install self hosted not in container environment?

### DISCUSSION THREAD

**Discussion by hexxamillion** [Budibase/budibase#14064] [Help] [answered]
I was wondering if it is possible to install budibase not in a container system but just on a bare system. It's not used for anything else and containers will be to complex for this setup. Just want to know if there is a way to just install on linux with postgres db.

**Comment by mike12345567** [comment, marked as answer]
Hi @hexxamillion Budibase requires a few different services to function (Redis, CouchDB, MinIO, nginx) which would be very difficult to configure exactly as Budibase needs on a bare-metal system. The single image version of Budibase (https://docs.budibase.com/docs/docker) is by far the quickest way to get setup and only requires Docker to be installed on the Linux host.

We don't provide any tooling for bare metal installations as it would be very difficult to maintain or to get any support for.
